api/rest/stat_series.md

```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
 - `unit`: The dcid of the preferred `unit` value.
 - `scaling_factor`: The preferred `scalingFactor` value.
 
-## GET Request
+## How to construct a request to the place statistics time series endpoint
 
-**Examples**
+### Step 1: Assembling the information you will need
 
-```bash
-curl 'https://api.datacommons.org/stat/series?place=geoId/06&stat_var=Count_Person_Male'
-```
+This endpoint requires two arguments and offers four additional optional arguments, as listed:
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/w32gmo68/8/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+ - `place`: For this parameter, you will need to specify the DCID (the unique ID assigned by Data Commons to each node in the graph) of the place you are interested in.
+ - `stat_var`: The statistical variable whose value you are interested in.
+
+In addition to these required properties, this endpoint also allows for other, optional arguments.
+
+  - `date`: If the property queried only takes on node values, you can use this argument to filter nodes in the response, ensuring the response only contains nodes with the specified type.
+
+  - `measurement_method`: You can specify this argument as `out` to indicate that you desire the response to only include nodes which are supercategories of the specified `DCIDs`, or `in` to only return nodes that are subcategories of the specified `DCIDs`. (For example, South America is a supercategory of Argentina, which in turn is a supercategory of Buenos Aires, as illustrated in Figure 1.)
+  
+  - `observation_period`: (≤ 500) Maximum number of values returned per node.
+
+  - `unit`: The unit of measurement.
+
+### Step 2: Creating the request
 
-## Success Response
+Since only the GET method is available for this endpoint, you will need to assemble the request via query parameters in the URL.
 
-### **Code**: `200 OK`
+## What to expect in the response
 
-**Response content example**
+Your response will always look like this:
 
 ```json
 {
-  "series": {
-    "2013": 18726468,
-    "2014": 18911519,
-    "2015": 19087135,
-    "2016": 19200970,
-    "2017": 19366579,
-    "2018": 19453769,
-    "2011": 18387718,
-    "2012": 18561020
-  }
+  "value": integer
 }
 ```
 
-## Error Response
+## Example requests and responses
 
-### **Code**: `400 Bad Request`
+### Example 1: Retrieve the count of men in the state of California.
```

Comment:
I think some of the jsfiddles are not for stat series. I'm also confused about the diff between the GET and CURL tabs. For stat series, we don't have POST, right?


@KilimAnnejaro I'll assume this section might have some pending changes. LMK when you're ready and I'll review again!

Reply:
Hi @tjann, thanks for these comments! As I mentioned in my message, there was a bug causing display issues here. I've fixed that (pending review) and you shouldn't see jsfiddles that are not relevant to this doc.